src/widgets/ProfileWidget/subcomponents/DisconnectedProfileWidget/DisconnectedProfileWidget.tsx

```diff
@@ -0,0 +1,12 @@
+import React, { FC } from 'react';
+
+interface DisconnectedProfileWidgetProps {
+  className?: string;
+}
+
+const DisconnectedProfileWidget: FC<DisconnectedProfileWidgetProps> = ({ className = '' }) => {
+  console.log('DisconnectedProfileWidget');
+  return <div className={`disconnected-profile-widget ${className}`}>DisconnectedProfileWidget</div>;
```

Comment:
```suggestion
  return <div className={`profile-widget ${className}`}>DisconnectedProfileWidget</div>;
```

The idea is that you use the same styling and components as the `ConnectedProfileWidget`.

Reply:
:heavy_check_mark: